### ISSUE TITLE
inkscape fails on nixpkgs-unstable

### ISSUE THREAD

**Issue by hedning** [NixOS/nixpkgs#40586]
## Issue description

Hydra log: https://hydra.nixos.org/build/74043642/log

~Did a bisect, which says that 2d4e30ccb028e2d0369460e2b9b3462a89bf954e was the first bad commit~ (the commit just happened to cause a rebuild triggering a non-deterministic failure):
```
git bisect start
# bad: [2d4e30ccb028e2d0369460e2b9b3462a89bf954e] graphite2: fix cross
git bisect bad 2d4e30ccb028e2d0369460e2b9b3462a89bf954e
# good: [183f56f2d62eabb3f60fb7417fdbdfb930d1da31] acpid: 2.0.28 -> 2.0.29
git bisect good 183f56f2d62eabb3f60fb7417fdbdfb930d1da31
# good: [188fdf5bd06de23bd289e2617bb01fb51928fdf9] Merge pull request #40189 from catern/contextvars
git bisect good 188fdf5bd06de23bd289e2617bb01fb51928fdf9
# good: [2f37fecfa1bf207d07b432ba19874f1569019232] Merge pull request #40359 from r-ryantm/auto-update/falkon
git bisect good 2f37fecfa1bf207d07b432ba19874f1569019232
# good: [59a6f939f83789c2f132bfda50fe0b1dc190f102] pythonPackages.prometheus_client: init at 0.2.0
git bisect good 59a6f939f83789c2f132bfda50fe0b1dc190f102
# good: [c4f5d559dad3f86071ddb8de02a9f3ed715d13b8] home-assistant: add prometheus component dependency
git bisect good c4f5d559dad3f86071ddb8de02a9f3ed715d13b8
# first bad commit: [2d4e30ccb028e2d0369460e2b9b3462a89bf954e] graphite2: fix cross
```

Not sure why that should break anything though, as the commit simply moves cmake from buildInputs to nativeBuildInputs in graphite2.

cc @globin

**Comment by matthewbauer** [comment]
I think it's a Make issue. We used to have CMake with Inkscape but then changed it to fix #39175. Hopefully we can switch back once #40046 is fixed.

Something to try is setting ```enableParallelBuilding = false;```

**Comment by hedning** [comment]
Ah yes, tried to actually build the «bad» commit and it  looks like a non-deterministic failure that just happened to get triggered when the graphite2 change caused a rebuild.

Tried to reproduce the failure locally a bunch of times without success (using `--cores 4`). Might be enough to just re-run the inkscape job manually until  #40139 hits master?

**Comment by hedning** [comment]
Should be fixed by #40717
